sirepo/package_data/static/js/sirepo-utils.js

```diff
@@ -11,6 +11,32 @@ class SirepoUtils {
         return s.charAt(0).toUpperCase() + s.slice(1);
     }
 
+    static orderOfMagnitude(val, binary=false) {
+        const MAGS = ['', 'k', 'M', 'G', 'T', 'P', 'E'];
+        let i = 0;
+        let v = 0;
+        const base = binary ? 1024 : 1000;
+        for (i = 0; i < MAGS.length; ++i) {
+            v = val / Math.pow(base, i);
```

Comment:
Doesn't this break if val is negative?

Reply:
[adbb0c2](https://github.com/radiasoft/sirepo/pull/4736/commits/adbb0c2fb917f892c1960a6c2cdea3bdcdeffe6c)